src/commands/prepare-release.yml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2018, salesforce.com, inc.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+
+# How to author commands: https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-commands
+description: >
+  Use standard-version to prepare a release (version bump, git tag, changelogs)
+
+parameters:
+  git_username:
+    description: github user name
+    default: Release Orb
+    type: string
+  git_email:
+    description: github user email
+    default: alm-cli@salesforce.com
+    type: string
+
+steps:
+  - jq/install
```

Comment:
It's interesting how you did `jq/install` but then `install-standard-version`. Should they be similar?

Reply:
@amphro We get `jq/install` from the jq orb: `https://circleci.com/orbs/registry/orb/circleci/jq` which is why it's namespaced like that